MIGRATION.md

```diff
@@ -0,0 +1,187 @@
+# Migration
+
+  * [Migrating from zeebe-full-helm](#migrating-from-zeebe-full-helm)
+  * [Migrating from zeebe-cluster-helm](#migrating-from-zeebe-cluster-helm)
+
+## Migrating from zeebe-full-helm
+
+If you're running an earlier `zeebe-full-helm` chart release, and you want to migrate to `ccsm-helm` you need to
+adjust your `values.yaml` file. The dependencies have been renamed, which means in the `values.yaml` you have to rename
+the objects/charts
+
```

Comment:
💡 I would say something like: "Since the dependencies have been renamed it is mandatory to update the objects/charts in there." 

I guess it makes the sentence a bit easier. - Just an "editorial" proposal. No must.